pkg/repository/v1/sqlcv1/tasks.sql

```diff
@@ -914,3 +914,53 @@ ANALYZE v1_task;
 
 -- name: AnalyzeV1TaskEvent :exec
 ANALYZE v1_task_event;
+
+-- name: CleanupV1TaskRuntime :one
+WITH trs as (
+    SELECT vtr.task_id, vtr.task_inserted_at, vtr.retry_count
+    FROM v1_task_runtime vtr
+    LEFT JOIN v1_task vt ON vtr.task_id = vt.id
+        AND vtr.task_inserted_at = vt.inserted_at
+    WHERE vt.id IS NULL
+    LIMIT sqlc.arg(batch_size)::int
```

Comment:
nit for consistency with other places:
```suggestion
    LIMIT @batchSize::int
```